build.gradle

```diff
@@ -38,17 +43,19 @@ task runTestNGTestCases(type: Test) {
 
     useTestNG()
 
-    testLogging {
-        events "started", "passed", "failed", "skipped"
-    }
+
+
+    //testLogging {
+    //    events "started", "passed", "failed", "skipped"
+    //}
 
     options {
-        suites(suites)
+        suites('src/test/resources/Suite1.xml')
```

Comment:
This is hardcoded
It should exist a way to execute the by suites something like this

```
gradle executeTC -Psuites=Suite1.xml,Suite2.xml
```